### PR TITLE
release/9.0.6

### PR DESCRIPTION
**Pull request by zservies**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -3,7 +3,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v9.0.5...HEAD)
+# [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v9.0.6...HEAD)
+
+# [v9.0.5](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v9.0.5...v9.0.6)
```
Comment (maxwellmooney13) [inline changelog.md:8]: Is this labeling right? There is a 9.0.5 underneath but I'm not sure if that was just a mistake from the past

Comment (jensengar) [reply changelog.md:8]: 🔥

File: src/client-private.ts [hunk 2]
```diff
@@ -34,6 +35,7 @@ export async function setupStreamingClient (this: GenesysCloudWebrtcSdk): Promis
     appVersion: originAppVersion || this.VERSION,
     appId: originAppId || this.logger.clientId,
     optOutOfWebrtcStatsTelemetry: optOutOfTelemetry,
+    customHeaders: customHeaders || undefined
```
Comment (jensengar) [inline src/client-private.ts:38]: is this undefined needed :trollface:

**Review by jensengar — APPROVED** [review]
Approved with max's changelog feedback.

File: changelog.md (hunk 1 above)
Comment (zservies) [reply changelog.md:8]: > Is this labeling right? There is a 9.0.5 underneath but I'm not sure if that was just a mistake from the past

oops, forgot to update the label.

**Review by maxwellmooney13 — APPROVED** [review]

**Pull request merged**